packages/dashboard/src/components/DataDesignerModal.tsx

```diff
@@ -121,15 +126,18 @@ export const DataDesignerModal: React.FC<DataDesignerModalProps> = ({ vizKey, ro
               }}
             />
           ) : (
-            <InputSelect
-              options={Object.keys(config.datasets[configureData.dataKey]?.data[0] || {})}
-              value={config.rows[rowIndex].multiVizColumn}
-              label='Multi-Visualization Column'
-              initial='--Select--'
-              fieldName=''
-              updateField={(section, subsection, fieldName, value) => setMultiVizColumn(value)}
-              required
-            />
+            <>
+              <InputSelect
+                options={Object.keys(config.datasets[configureData.dataKey]?.data[0] || {})}
+                value={config.rows[rowIndex].multiVizColumn}
+                label='Multi-Visualization Column'
+                initial='--Select--'
+                fieldName=''
+                updateField={(section, subsection, fieldName, value) => setMultiVizColumn(value)}
+                required
+              />
+              <CheckBox value={config.rows[rowIndex].expandCollapseAllButtons} label=' Add Expand/Collapse All buttons' fieldName='' updateField={(section, subsection, fieldName, value) => setExpandCollapseAllButtons(value)} />

```

Comment:
Is fieldname needed here?

Reply:
By removing the fieldName this causes a TypeScript error since it is required

Reply:
I believe that can be resolved by adding a default value for fieldName on the checkbox component `fieldName = ''`